tests/coverage.sh

```diff
@@ -16,7 +16,8 @@ RUN echo 'deb http://deb.debian.org/debian stretch main' >> /etc/apt/sources.lis
     && (apt-get update || true) \
     && apt-get install -y git
 
-ADD base-template.sh install.sh uninstall.sh cli.sh .githooks/README.md /var/lib/githooks/
```

Comment:
Why did this have to change?

Reply:
Oh, think it was a copy paste from exec-test.sh
Now its the same...